.babelrc

```diff
@@ -22,6 +22,8 @@
         ]
       }
     ],
+    "transform-async-to-generator",
+    "transform-async-generator-functions",
```

Comment:
I don't understand why these two are needed, can you provide an explanation to it please? In the PR description I can read:
`Add some more transformations to be performed by Babel, to support async/await`

But still, it's not evident to me _where_  async/await is being used. Is this in preparation for an upcoming PR, maybe?

Reply:
Good question! Here's what happens: by trying to use the `src` version of the files in the GB packages, we by pass the step where the GB build transpiles them to ES5. So, we need to help Babel compile those for us.

The `async/await` usage is actually in the GB packages, not the RN app. See https://github.com/WordPress/gutenberg/blob/ccbb54b4de3f0c471fa938faecb6e52c166f3991/packages/data/src/index.js#L175. For example, comment out the `"transform-async-generator-functions"` line in our Babel configuration and see the error that is emitted. It will be in code inside the GB `data` package.

Let me know if this answers your question. 👍

Reply:
Thanks, I'd not find that myself other than by commenting that out as suggested